data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Original: </p><p>... you can separate the start time column ...</p><p> </p><p>Proposed:</p><p>... you can separate the <strong>date</strong> column ...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Also a great catch</p>